mlflow/entities/span.py

```diff
@@ -154,6 +154,12 @@ def events(self) -> List[SpanEvent]:
             for event in self._span.events
         ]
 
+    def __repr__(self):
+        return (
+            f'{type(self).__name__}(name="{self.name}", request_id={self.request_id}, '
```

Comment:
```suggestion
            f'{type(self).__name__}(name={self.name!r}, request_id={self.request_id}, '
```

Reply:
Can we also include `status`?

Reply:
Hmm not sure how useful it is, given that the status is mostly OK. Maybe we can wait the UI to render status first?🤔

Reply:
Got it, sounds good!